extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationService.js

```diff
@@ -177,8 +180,31 @@ oppia.factory('ItemSelectionInputValidationService', [
                       'rule must include at least 2 options.')
                   });
                 }
+              } else if (rule.type === 'Equals') {
+                if (minAllowedCount > ruleInputs.length ||
+                  maxAllowedCount < ruleInputs.length) {
+                  warningsList.push({
+                    type: WARNING_TYPES.ERROR,
+                    message: (
+                      'In answer group ' + (answerIndex + 1) + ', ' +
```

Comment:
'In answer group ' + (answerIndex + 1) + ', ' +
'rule ' + (ruleIndex + 1) + ', number of correct options' +
'in the "Equals" rule should be between the minimum and ' +
'maximum allowed selection count.'

Maybe it would also make sense to put in what the minimum and maximum is in the error message.

Reply:
Hi @nithusha21! 
>If 0 items are selected, can the user submit the answer?  

I feel we should allow 0 items to be selected. I think it can be used to test a user's understanding, and the user would know if 0 options can be selected. (in interaction,   _Please select at least one choice._ message is shown to user).

And I changed the error message, I agree its better to show the minimum and maximum in error message.

Did all the changes. Please take a look